style.css

```diff
@@ -7,124 +7,158 @@
   font-family: "Poppins", sans-serif;
 }
 
-header {
+body {
+  overflow-x: hidden;
+}
+
+.top-nav {
   display: flex;
-  width: 375px;
+  width: 100%;
   height: 48px;
   padding: 4px 24px 4px 12px;
-  align-items: flex-start;
-  gap: 199px;
-  flex-shrink: 0;
+  justify-content: space-between;
 }
 
 .logo {
-  display: flex;
   padding: 10px 12px;
-  align-items: center;
   color: #6070ff;
   text-align: center;
-  font-size: 18px;
+  font-size: 1.125rem;
   font-style: normal;
   font-weight: 700;
-  line-height: 20px;
+  line-height: 1.25rem;
+  transition: all 1s ease-in-out;
+}
+
+.logo:hover {
+  transform: rotate(180deg);
+}
+
+.nav-links-container {
+  display: none;
 }
 
 .Hamburger-menu {
-  display: flex;
-  padding: 8px;
-  align-items: flex-start;
+  padding: 0.5rem;
   border-radius: 8px;
   width: 24px;
   height: 24px;
 }
 
 .hero {
-  background-image: url(/images/background-shapes.png);
-  display: inline-flex;
-  padding: 114px 24px;
+  background-image: url(./images/Header\ bg@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  padding: 20% 10%;
   flex-direction: column;
-  align-items: flex-start;
-  gap: 12px;
-  width: 375px;
+  text-align: start;
+  gap: 0.75rem;
+  width: 100%;
+  height: auto;
```

Comment:
Kindly consider to add the border-radius to look like the Figma design 😁 
`    border-bottom-left-radius: ?rem;
`

![Screenshot 2023-11-30 213132](https://github.com/Frederick-Ye/portfolio-project/assets/102011195/0499315f-0bd6-435b-b088-b327e66e7b1b)